### PR TITLE
<AvatarGroup/> - add avatarGroup component to cheatsheet

### PR DESCRIPTION
**Pull request by roibWix**
### 🔦 Summary

...

### ✅ Checklist

- [ ] 👨‍💻 API change is approved by the UI Infra Developers 
- [ ] 👨‍🎨 UX change is approved by the Design System UX 
- 📚 Change is documented
  - [ ] Story
  - [ ] API description
  - [ ] Cheatsheet
  - [ ] Other (explain)
- 🔬 Change is tested
  - [ ] Component
  - [ ] Visual test

### CODE REVIEW

**Comment by WixBuildServer** [comment]
### 🤖 Hi, I'm #Janet!

  A deploy preview of your PR was successfully released 🖼.

  Project Name: **wix-ui-tpa**

  You can find it under one of the following URLs:
  * https://bo.wix.com/pages/wix-ui-tpa.pr_6830/

**Comment by WixBuildServer** [comment]
### 🤖 Hi, I'm #Janet!

  A deploy preview of your PR was successfully released 🖼.

  Project Name: **wix-style-react**

  You can find it under one of the following URLs:
  * https://preview.wix-style-react.com/pr_6830

File: packages/wix-style-react/stories/Introduction/Cheatsheet/componentsFamilies/OtherFamily.js [hunk 1]
```diff
@@ -92,6 +93,65 @@ const AvatarExample = () => {
...
+  ];
+  const symbol = otherSymbols.avatarGroup;
+  const components = otherSymbolsToComponents[symbol];
+  const singleComponentProps = {
+    name: symbol,
+    componentsNames: createLinkedComponentsNames(components),
+  };
+  return (
+    <SingleComponentSideBySide {...singleComponentProps}>
+      <Layout cols={7} justifyItems="center" alignItems="center">
```
Comment (TalZa1810) [inline packages/wix-style-react/stories/Introduction/Cheatsheet/componentsFamilies/OtherFamily.js:148]: why do we need the `<Layout/>` here?

File: packages/wix-style-react/stories/symbolsComponentsMapping/symbols.js [hunk 2]
```diff
@@ -147,16 +147,17 @@ export const pickerSymbols = {
...
-  badgeSelect: '11.3 Badge Select',
-  counterBadge: '11.4 Counter Badge',
-  tag: '11.5 Tag',
-  loader: '11.6 Loader',
-  linearProgressBar: '11.7 Linear Progress Bar',
-  circularProgressBar: '11.8 Circular Progress Bar',
-  skeleton: '11.9 Skeleton',
-  image: '11.10 Image',
-  palette: '11.11 Palette',
+  avatarGroup: '11.2 AvatarGroup',
```
Comment (TalZa1810) [inline packages/wix-style-react/stories/symbolsComponentsMapping/symbols.js:150]:
```suggestion
  avatarGroup: '11.2 Avatar Group',
```

**Review by TalZa1810 — APPROVED** [review]
just few notes to fix. looks good overall.

File: packages/wix-style-react/stories/Introduction/Cheatsheet/componentsFamilies/OtherFamily.js (hunk 1 above)
Comment (roibWix) [reply packages/wix-style-react/stories/Introduction/Cheatsheet/componentsFamilies/OtherFamily.js:148]: no need

**Pull request merged**